it/uk/gov/hmrc/transitmovementsrouter/connectors/PersistenceConnectorSpec.scala

```diff
@@ -112,6 +117,15 @@ class PersistenceConnectorSpec
       )
   )
 
+  def stubForPatchMessageStatus(codeToReturn: Int, body: Option[String] = None) = server.stubFor(
+    patch(
+      urlEqualTo(uriStatus)
+    ).willReturn(
+      if (body.isEmpty) aResponse().withStatus(codeToReturn)
+      else aResponse().withStatus(codeToReturn)
```

Comment:
Pease use ``` else aResponse().withStatus(codeToReturn).withBody(body.get) ``` to test with error response with body

Reply:
That doesn't make sense, considering that the `body` variable is for the request, not the response.

Reply:
Yes, this `if` statement is pointless here.